client/my-sites/upgrades/checkout/thank-you.jsx

```diff
@@ -63,6 +66,14 @@ var CheckoutThankYou = React.createClass( {
 	},
 
 	componentDidMount: function() {
+		var selectedSite = this.props.lastTransaction.selectedSite;
+		this.props.refreshSitePlans( selectedSite.ID );
```

Comment:
Do we need to refresh these for all purchases, or just when the user checks out with a trial/plan?